packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -196,6 +254,116 @@ - (FlutterError *_Nullable)onListenWithArguments:(id _Nullable)arguments
   _eventSink = events;
   return nil;
 }
+- (void)startRecordingVideoAtPath:(NSString *)path result:(FlutterResult)result {
+  if (!_isAudioSetup) {
+    [self setUpCaptureSessionForAudio];
+  }
+
+  if (!_isRecording) {
+    NSLog(@"start video recording...");
+    if (![self setupWriterForPath:path]) {
+      NSLog(@"Setup Writer Failed");
+      return;
+    }
+    [_captureSession stopRunning];
+    _isRecording = YES;
+    [_captureSession startRunning];
+  }
+  // NSLog(@"Saved: %@ ", path );
+}
+- (void)stopRecordingVideo {
+  if (_isRecording)
+
+  {
+    _isRecording = NO;
+    [_captureSession stopRunning];
```

Comment:
I think it i possible to rewire the capturesession without stopping it.
That would give much less jank.

Reply:
Yes, this is possible.